grpc/src/main/java/io/stargate/grpc/service/CreateTableHandler.java

```diff
@@ -35,15 +36,15 @@
       CqlTableCreate createTable,
       Persistence.Connection connection,
       Persistence persistence,
-      QueryBuilder queryBuilder,
+      TypedValue.Codec valueCodec,
```

Comment:
The query builder has a static view of the schema, so it has to be built at the last moment.

Reply:
So basically `valueCodec` needs to be passed to allow us to construct `QueryBuilder` when needed?

Reply:
Yes.
We could rebuild it every time too, but that seems inefficient since it can be cached.